### PR TITLE
[fix] failed to save checkpoints in VideoLearner.fit()

### PR DESCRIPTION
**Pull request by xueeinstein**
### Description

Fix format bug of path string.

Failed to save checkpoints while training the model using `video_learner.fit(save_model=True)`

### Related Issues
None

### Checklist:

- [ x] I have followed the [contribution guidelines](../CONTRIBUTING.md) and code style for this project.
- [x ] I have added tests covering my contributions.
- [ ] I have updated the documentation accordingly.
- [ ] This PR is being made to `staging` and not `master`
- [ ] I will squash merge this PR into `staging`

### CODE REVIEW

File: utils_cv/action_recognition/model.py [hunk 1]
```diff
@@ -273,7 +273,7 @@ def fit(
                 self.save(
                     os.path.join(
                         model_dir,
-                        "{model_name}_{self.epoch}.pt".format(
+                        "{model_name}_{epoch}.pt".format(
```
Comment (PatrickBue) [inline utils_cv/action_recognition/model.py:276]: Should the line be as suggested:
"{model_name}_{epoch}.

Or should it rather be like below ("epoch" does not seem to be defined)
"{model_name}_{e}

Comment (xueeinstein) [reply utils_cv/action_recognition/model.py:276]: Actually `epoch` is defined by the parameter of `format`. See L277. This fix works perfectly for me.

Comment (PatrickBue) [reply utils_cv/action_recognition/model.py:276]: Thanks, I see it now!

**Comment by PatrickBue** [comment]
Thanks for making this fix. I would approve, but the PR is made to master. Can you please make a PR to the "staging" branch instead? As per our contribution guidelines we only make code changes to staging, and update master in regular intervals.

**Comment by xueeinstein** [comment]
@PatrickBue Sure. I just close this PR and open a new one, [#613 ](https://github.com/microsoft/computervision-recipes/pull/613)

**Pull request closed without merging**